File: JAVA/Codigo en Java/src/Persona.java
public class Persona {

    private String nombre;
    private String profesion;
    private String sexo;
    private int edad;

    public Persona(){
        this.setEdad(0);
    }


    public Persona(String nombre, String profesion, String sexo, int edad) {
        this.nombre = nombre;
        this.profesion = profesion;
        this.sexo = sexo;
        this.edad = edad;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getProfesion() {
        return profesion;
    }

    public void setProfesion(String profesion) {
        this.profesion = profesion;
    }

    public String getSexo() {
        return sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }


    public void validarValor(double edad) {
        if (edad < 0) {
            throw new IllegalArgumentException("La edad No puede ser negativa");
        }
    }

}

File: Persona.py
class Persona:
    pass

    def __init__(self, nombre, profesion, sexo, edad):
        self.__nombre = nombre
        self.__profesion = profesion
        self.__sexo = sexo
        self.__edad = edad


    @property
    def nombre(self):
        return self.__nombre

    @nombre.setter
    def nombre(self, nombre):
        self.__nombre = nombre

    @property
    def profesion(self):
        return self.__profesion

    @profesion.setter
    def profesion(self, profesion):
        self.__profesion = profesion

    @property
    def sexo(self):
        return self.__sexo

    @sexo.setter
    def sexo(self, sexo):
        self.__sexo = sexo

    @property
    def edad(self):
        return self.__edad

    @edad.setter
    def edad(self, edad):
        self.__edad = edad

    def validador(self, edad):
        if edad < 0:
            raise Exception("Edad no puede ser menor a 0")


    def __str__(self):
        return "Nombre: " + str(self.__nombre) + "  Profesion: " + str(self.__profesion) + "  Sexo: " + str(self.__sexo) + " Edad: " + str(self.__edad)


p1 = Persona("Freddy", "Estudiante", "Masculino", 2)

p1.validador(2)

print(p1)